flow-report/src/i18n/i18n.tsx

```diff
@@ -31,23 +31,23 @@ function useLhrLocale() {
   };
 }
 
-export function useI18n() {
+function useI18n() {
   return useContext(I18nContext);
 }
 
-export function useLocalizedStrings() {
+function useLocalizedStrings() {
   const i18n = useI18n();
   return i18n.strings;
 }
 
-export function useStringFormatter() {
+function useStringFormatter() {
   const {locale} = useLhrLocale();
   return (str: string, values?: Record<string, string|number>) => {
     return formatMessage(str, values, locale);
   };
 }
 
-export const I18nProvider: FunctionComponent = ({children}) => {
+const I18nProvider: FunctionComponent = ({children}) => {
```

Comment:
Now that `Util.i18n` is a true static member, can we remove the `@ts-ignore` on L69?